roles/reproducer/tasks/ci_data.yml

```diff
@@ -60,7 +60,11 @@
       ansible.builtin.set_fact:
         cacheable: true
         ci_job_networking: "{{ zuul_inventory.all.hosts.crc.crc_ci_bootstrap_networking }}"
-        use_content_provider: >-
```

Comment:
I don't get it - why changing the name, and having 2 facts with the exact same value?

Reply:
I still don't get this too.
if `zuul_params.content_provider_registry_ip`  is defined, both `openstack_content_provider` and `operator_content_provider` will be set to `true`

Reply:
yeah, sorry. Updating this in a few mins

Reply:
ah, now I get it :)